packages/transport/test/modules/relationships/PeerDeletionInfo.test.ts

```diff
@@ -0,0 +1,16 @@
+import { CoreDate } from "@nmshd/core-types";
+import { PeerDeletionInfo, PeerDeletionStatus } from "../../../src";
+
+describe("PeerDeletionInfo", () => {
+    test("PeerDeletionInfo default values", () => {
```

Comment:
this doesn't match our test naming and does not explain what should be tested here. I you would have written a proper test name you would have noticed that this test should be splitted in two tests.